extensions/signal.sigmf-ext.md

```diff
@@ -0,0 +1,285 @@
+
+The field of communications is vast, and there may be communications systems that cannot be described using the names and fields described in this extension. If you need additional or different fields to describe a system, create a new extension that adds the necessary fields to the `signal` namespace and/or submit the new fields to be upstreamed into this specification.
+
+#### The `emission` Object
+
+`emission` objects contain name/value pairs that describe the modulations of communications systems. These objects MAY contain the following pairs:
+
+|name|required|type|description|
+|----|--------------|-------|----------|
+|`label`|false|string|A high-level shortform description useful to human readers (e.g., "wifi", "LTE")|
+|`detail`|false|[Detail](signal.sigmf-ext.md#the-detail-object)|Emission details (standard, modulation, etc.)|
+|`emitter`|false|[Emitter](signal.sigmf-ext.md#the-emitter-object)|Emitter details (manufacturer, geo coordinates, etc.)|
+
+##### The Emitter Object
+
+|name|required|type|description|
+|----|--------|----|-----------|
+|`seid`|false|uint|Unique ID of the emitter|
+|`manufacturer`|false|string|Manufacterer of the hardware used to emit the signal|
+|`latitude`|false|float|Latitude (decimal degrees)|
+|`longitude`|false|float|Longitude (decimal degrees)|
+
+##### The Detail Object
+
+|name|required|type|description|
+|----|--------|----|-----------|
+|`type`|false|string|[type](signal.sigmf-ext.md#the-type-field)|
+|`class`|false|string|[class](signal.sigmf-ext.md#the-class-field)|
+|`standard`|false|string|Communication standard (e.g., 802.11ac)|
+|`carrier_variant`|false|string|[carrier variant](signal.sigmf-ext.md#the-carrier_variant-field)|
+|`symbol_variant`|false|string|[symbol variant](signal.sigmf-ext.md#the-symbol_variant-field)|
+|`order`|false|uint|[order](signal.sigmf-ext.md#the-order-field)|
+|`duplexing`|false|string|[duplexing](signal.sigmf-ext.md#the-duplexing-field)|
+|`multiplexing`|false|string|[multiplexing](signal.sigmf-ext.md#the-multiplexing-field)|
+|`multiple_access`|false|string|[multiple access](signal.sigmf-ext.md#the-multiple_access-field)|
+|`spreading`|false|string|[spreading](signal.sigmf-ext.md#the-spreading-field)|
+|`bandwidth`|false|float|[bandwidth](signal.sigmf-ext.md#the-bandwidth-field)|
+|`channel`|false|uint|[channel](signal.sigmf-ext.md#the-channel-field)|
+|`class_variant`|false|string|[class variant](signal.sigmf-ext.md#the-class_variant-field)|
+
+###### The `type` Field
+
+The `type` field can have the following values:
+
+|value|description|
+|----|-------|
+|`analog`|analog modulation scheme|
+|`digital`|digital modulation scheme|
+
+###### The `class` Field
+
+The `class` field can have the following values:
+
+|value|description|
+|----|-------|
+|`am`|(analog) amplitude modulation|
+|`fm`|(analog) frequency modulation|
+|`pm`|(analog) phase modulation|
+|`ssb`|(analog) single side-band|
+|`dsb`|(analog) dual side-band|
+|`vsb`|(analog) vestigal side-band|
```

Comment:
This is minor, but VSB can also used with digital modulations (see: ATSC)

Reply:
Interesting, I didn't know that! @Atheatos, want to make this `(analog / digital)` for completeness?